internal/internal_workflow_client.go

```diff
@@ -1170,3 +950,273 @@ func serializeSearchAttributes(input map[string]interface{}) (*commonpb.SearchAt
 	}
 	return &commonpb.SearchAttributes{IndexedFields: attr}, nil
 }
+
+type workflowClientInterceptor struct{ client *WorkflowClient }
```

Comment:
Sorry, I might miss something obvious. Is this new or old code that got moved from somewhere?
If it's new what's the purpose?

Reply:
This is a new type. Client interceptors are a new thing. This is the implementation of last outbound interceptor in the client chain.